### PR TITLE
Added logging to console for debugging. 

### PR DESCRIPTION
**Pull request by sontheimer**
Logging placeholders in the same style as previous versions added to
- python/Interscale_hub/InterscaleHub.py
- python/Interscale_hub/pivot.py 
- python/placeholders/Simulation_mock.py 

Some log formatting to remove log error messages.

### CODE REVIEW

File: python/Interscale_hub/pivot.py [hunk 1]
```diff
@@ -40,8 +40,13 @@ def __init__(self, param, comm_receiver, comm_sender, databuffer):
         '''
         
         # TODO: logger placeholder for testing
+        import sys
```
Comment (w-klijn) [inline python/Interscale_hub/pivot.py:43]: import in functions are not best practice.

Just add at the start of the file

File: python/placeholders/Simulation_mock.py [hunk 2]
```diff
@@ -82,7 +82,13 @@ def __init__(self,path):
         '''
         
         # TODO: logger placeholder for testing
+        import sys
```
Comment (w-klijn) [inline python/placeholders/Simulation_mock.py:85]: import in functions are not best practice.

Just add at the start of the file

File: python/Interscale_hub/pivot.py [hunk 3]
```diff
@@ -40,8 +40,13 @@ def __init__(self, param, comm_receiver, comm_sender, databuffer):
...
         self.__logger = logging.getLogger(__name__)
-        self.__logger.info("Initialise...")
+        handler = logging.StreamHandler(sys.stdout)
+        formatter = logging.Formatter('%(asctime)s - %(name)s - %(levelname)s - %(message)s')
```
Comment (w-klijn) [inline python/Interscale_hub/pivot.py:46]: Isn't this something that is configurable? It feels somewhat repeated to have the same rather complex line on multiple locations

File: python/placeholders/Simulation_mock.py [hunk 4]
```diff
@@ -196,7 +202,13 @@ def __init__(self,path):
...
```
Comment (w-klijn) [inline python/placeholders/Simulation_mock.py:205]: Move to start file

File: python/tests/app_interscalehub.py [hunk 5]
```diff
@@ -21,7 +21,7 @@ def run_wrapper(args):
     # 1 --> nest to Tvb
     # 2 --> tvb to nest
     param = Parameter()
-    direction = int(args[1])
+    direction = int(args)
```
Comment (w-klijn) [inline python/tests/app_interscalehub.py:24]: I would put a try catch around this. int() fails hard

**Review by w-klijn — CHANGES_REQUESTED** [review]
Looks ok to me.

Better to move all import to the start of the files.

Some small remarks on other location. That one is taste

**Review by mfahdaz — COMMENTED** [review]
Logging will help in debugging while connecting InterscaleHub with other Components.

NOTE: Later, it is going to be configured with the settings from global configurations file for a uniform log settings,

File: python/tests/app_interscalehub.py (hunk 5 above)
Comment (sontheimer) [reply python/tests/app_interscalehub.py:24]: I add a NOTE. For the moment this is a temporary solution and will be changed in the next iteration (proper interface for steering).

**Comment by sontheimer** [comment]
- moved import statements to the start of the file (PEP8).
- All three locations are duplicate code but marked with a TODO for proper logging -> included in the next iteration when we use the cosim manager (incl. logging).

**Pull request merged**